charts/k8tz/values.yaml

```diff
@@ -12,6 +12,12 @@ verbose: false
 webhook:
   failurePolicy: Fail
 
+  certManager:
```

Comment:
values must be documented in the chart's [README](https://github.com/k8tz/k8tz/blob/master/charts/k8tz/README.md#values)